Makefile

```
@@ -10,14 +10,18 @@ install-deps: ## Install dependencies
+
 .PHONY: test-coverage
-test-coverage: ## Run tests with coverage
+test-coverage: generate ## Run tests with coverage
 	@go test -v -coverprofile=coverage.out ./... -count=1 2>&1 | tee /dev/stderr | go-junit-report -set-exit-code > report.xml
 
 .PHONY: test
-test: ## Run tests
+test: generate ## Run tests
 	@go test -v ./... -count=1
 
 .PHONY: build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)